modules/core/src/main/scala/org/scalasteward/core/edit/update/VersionPositionScanner.scala

```diff
@@ -44,8 +44,9 @@ object VersionPositionScanner {
     }
 
   private def sbtModuleIdRegex(version: Version): Regex = {
+    val ident = """[^\s]+"""
```

Comment:
I locally tried to simplify
```suggestion
    val ident = """\S+"""
```
but it is slower

Reply:
Oh, that's interesting and strange given that `[^\s]` looks like the exact definition of `\S`.

Reply:
Some more runs lead to no measurable difference. Which is what to expect 😄